equational_theories/ManuallyProved/Equation1648.lean

```diff
@@ -19,3 +20,19 @@ theorem Equation1648_not_implies_Equation206 : ∃ (G: Type) (_: Magma G), Equat
       split <;> simp_all <;> split <;> split <;> simp_all <;> linarith
   · use 0,-1
     simp [instMagmaInt, sign]
+
+theorem Equation1648_not_implies_Equation151 : ∃ (G: Type) (_: Magma G),
+  Equation1648 G ∧ ¬ Equation151 G := by
+  let instMagmaInt : Magma ℤ := {
+    op := fun x y => if x > y then x + 1 else x - 1
+  }
+  use ℤ, instMagmaInt
+  simp [Equation1648, Equation151]
+
+  constructor
+  · intro x y
+    simp [instMagmaInt]
+    split <;> simp <;> split <;> simp_all <;> linarith
+  · simp [instMagmaInt]
```

Comment:
Please use `simp only` here too.

Reply:
Same as above. `simp` is applying more useful simplifications here.